### ISSUE TITLE
1. The title should hint at what the paper / repository are all about.

### ISSUE THREAD

**Issue by vsoch** [gasparl/citapp_pc#1]
My first reaction is that I don't know what a "Concealed Information Test" is. I do have a gist of what a reaction time is. Is there a way to better describe the work, and simplify the title to be stupidly obvious in layman's terms? I would also want to see this title in the repository description (if you are new to Github, it's the little spot of text next to the repository name that you can click to edit.)
Application is also very general. Is it a desktop application? A mobile application? A web application? Something else?

Review: https://github.com/openjournals/joss-reviews/issues/1179

**Comment by gasparl** [comment]
I renamed it as "CITapp - a response time-based Concealed Information Test lie detector web application".
(Even if not the best from the layman perspective, I would keep the "Concealed Information Test" part in the title, because it is simply the proper name of the method, already used in hundreds of publications.)
